integration/sql/impl/tests/table_lineage/tests_select.rs

```diff
@@ -86,14 +86,13 @@ fn select_left_join() {
 #[test]
 fn select_bigquery_excaping() {
```

Comment:
e`X`caping or escaping? I know it's not modified by your PR but anyway would be nice if u had a second.

Reply:
Fixed.